src/components/submission/feed-entry.vue

```diff
@@ -215,7 +230,7 @@ export default {
       */
       "create": "Submitted by {name}",
       "entity": {
-        "create": "Created Entity {label} in {dataset} Dataset",
+        "create": "Created Entity {label} in Dataset {dataset}",
```

Comment:
I see what you mean about the inconsistency. I think it works this way only because {dataset} is a single word, so "{dataset} Dataset" becomes almost like a compound noun.